insights/parsers/ip.py

```diff
@@ -119,6 +119,10 @@ def parse_link(line, d):
         d["peer"] = split_content[3]
     elif len(split_content) >= 2:
         d["mac"] = split_content[1]
+        for item in split_content:
+            if item == "promiscuity":
+                index = split_content.index(item)
+                d["promiscuity_mode"] = split_content[index + 1]
```

Comment:
How about 
```suggestion
        if "promiscuity" in split_content":
            d["promiscuity"] = split_content[split_content.index('promiscuity') + 1]
```

I suggest modifying the name of the key to `promiscuity` as it is in the line, in case, this parser would be enhanced by other utility helpers automatically.

Reply:
Updated according to suggestion.